src/components/Tabset/styled/buttonIcon.js

```diff
@@ -1,8 +1,18 @@
 import styled from 'styled-components';
 import ButtonIcon from '../../ButtonIcon';
 import { COLOR_WHITE, COLOR_GRAY_2 } from '../../../styles/colors';
+import getTheme from '../../../styles/helpers/getTheme';
+
+const StyledButtonIcon = styled(ButtonIcon).attrs(props => {
+    const brandMainColor = getTheme(props).palette.brand.main;
+    return { brandMainColor };
+})`
+    color: ${props => props.brandMainColor};
+
+    &:hover {
+        color: ${props => props.brandMainColor};
```

Comment:
Because of the changes in the icons when you set:
```fill="currentColor"```
then below when disabled use
```color: ${COLOR_GRAY_2};```
instead of the current introspection
```
> svg {
    fill: ${COLOR_GRAY_2};
}
```